app/components/pretty-json.js

```diff
@@ -15,7 +15,16 @@ export default Component.extend({
 
   didInsertElement() {
     this._super(...arguments);
+    this.$().append(this.getRenderData());
+  },
+
+  didUpdateAttrs() {
+    this._super(...arguments);
+    this.$().empty().append(this.getRenderData());
+  },
```

Comment:
Combine didInsertElement and didUpdateAttrs to didRender which is called on both init-render and re-render?